"openssl 1.0.0b fails to install when compiled with the no-hw option"
http://rt.openssl.org/Ticket/Display.html?id=2384&user=guest&pass=guest

On part is extracted to Configure.patch

diff -u -r ../openssl-1.0.1f/engines/Makefile ./engines/Makefile
--- ../openssl-1.0.1f/engines/Makefile	2014-01-06 15:36:06.000000000 +0100
+++ ./engines/Makefile	2014-03-02 22:04:09.000000000 +0100
@@ -113,6 +113,7 @@
 			  pfx=lib; \
 			  if [ "$(PLATFORM)" != "Cygwin" ]; then \
 				case "$(CFLAGS)" in \
+				*OPENSSL_NO_HW*)	echo ... skipping install OPENSSL_NO_HW defined; continue;; \
 				*DSO_BEOS*)	sfx=".so";;	\
 				*DSO_DLFCN*)	sfx=`expr "$(SHLIB_EXT)" : '.*\(\.[a-z][a-z]*\)' \| ".so"`;;	\
 				*DSO_DL*)	sfx=".sl";;	\
diff -u -r ../openssl-1.0.1f/engines/ccgost/Makefile ./engines/ccgost/Makefile
--- ../openssl-1.0.1f/engines/ccgost/Makefile	2014-01-06 15:36:06.000000000 +0100
+++ ./engines/ccgost/Makefile	2014-03-02 22:04:09.000000000 +0100
@@ -47,6 +47,7 @@
 		pfx=lib; \
 		if [ "$(PLATFORM)" != "Cygwin" ]; then \
 			case "$(CFLAGS)" in \
+			*OPENSSL_NO_HW*)	echo ... skipping install OPENSSL_NO_HW defined; exit;; \
 			*DSO_BEOS*) sfx=".so";; \
 			*DSO_DLFCN*) sfx=`expr "$(SHLIB_EXT)" : '.*\(\.[a-z][a-z]*\)' \| ".so"`;; \
 			*DSO_DL*) sfx=".sl";; \
